### PR TITLE
feat: simplify the input interface

### PR DESCRIPTION
**Pull request by kettanaito**
This pull request reflects on a larger discussion about the GitHub Action. 

## Changes

- Removes all granular action inputs. Instead, a single `command` input should be used.
- Removes the `report` action. I'd not have any implicit behaviors, especially when they deviate from the raw CLI experience. The users can use the `--output` flag to generate the report and access it in the file system since they know the path already. 
- Removes usage examples from `README.md`. Directs to the appropriate page in the docs. We shouldn't duplicate these examples, let them live in a single place. Easy to find, easy to manage. 
- Adjusts the test workflow.
- Adds a new test to ensure that the generated report can be accessed by other steps of the job (e.g. to manipulate it, upload it as an artifact, etc).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,153 +11,19 @@ Official GitHub Action for running load tests with <a href="https://artillery.io
 
 The `artilleryio/action-run` action supports a subset of the [`run` command options](https://www.artillery.io/docs/reference/cli/run#options) of our CLI.
 
-### `test`
+### `command`
 
-A test script path.
+A path to the test script to run.
```
Comment (bernardobridge) [inline README.md:16]: This should be updated no?

e.g. `The Artillery CLI command to run.`

**Review by bernardobridge — CHANGES_REQUESTED** [review]

File: entrypoint.sh [hunk 2]
```diff
@@ -1,14 +1,7 @@
...
 # Run the tests.
-/home/node/artillery/bin/run run $1 \
-  --output "$OUTPUT" \
-  ${CONFIG:+-c $CONFIG}
-  ${ENVIRONMENT:+-e $ENVIRONMENT}
-  ${TARGET:+-t $TARGET}
-  ${DOTENV:+--dotenv $DOTENV}
-  ${INSECURE:+--insecure}
-  ${QUIET:+-q}
+/home/node/artillery/bin/run "./artillery.report.yml" $1
```
[the remark this answers is not in the document]
Comment (kettanaito) [reply entrypoint.sh:4]: I was thinking about this and decided that it's not a good idea to introduce this implicit behavior for the action alone. The regular `run` command doesn't produce any implicit reports, neither should the action. Let the usage be identical, no matter if you're using an action or tapping into the CLI directly.

File: entrypoint.sh [hunk 3]
```diff
@@ -1,14 +1,7 @@
...
 
 # Write outputs for the GitHub Action.
 echo "report=$OUTPUT" >> $GITHUB_OUTPUT
```
[the remark this answers is not in the document]
Comment (kettanaito) [reply entrypoint.sh:7]: I should actually remove this altogether. The action doesn't set any outputs anymore. The users can still generate the output as they'd normally would working with the CLI, and access that report on the file system level.

File: entrypoint.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (bernardobridge) [reply entrypoint.sh:4]: How would a user extract a report from the Github Action though? We might want to showcase this in an example

File: README.md (hunk 1 above)
Comment (kettanaito) [reply README.md]: Done!

File: entrypoint.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (kettanaito) [reply entrypoint.sh]: Removed this.

File: entrypoint.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (kettanaito) [reply entrypoint.sh:4]: What do you mean by "extract"? 

One of the ways they may want to operate with the generated report is by uploading it as an artifact on the workflow to observe after the run. 

```yml
- uses: artilleryio/action-cli@v1
  with:
    command: run test.yml --output report.json

- uses: actions/upload-artifact@v3
  with:
    name: artillery-report
    path: report.json
```

I believe we have this example in the docs. If we don't, I will add it once we publish the action.

[the remark this answers is not in the document]
Comment (kettanaito) [reply entrypoint.sh:4]: One good point from this is we should add an automated test to ensure the report written in the Docker image is actually available outside of the step 💯 Will add that.

[the remark this answers is not in the document]
Comment (kettanaito) [reply entrypoint.sh]: Added a basic test that the generated report exists in the file system. Waiting for it to pass...

**Review by bernardobridge — APPROVED** [review]

**Review by hassy — APPROVED** [review]
:shipit:

**Pull request merged**